build.sbt

```diff
@@ -4313,6 +4311,7 @@ lazy val `os-environment` =
       ),
       Compile / internalModuleDependencies ++= Seq(
         (`engine-common` / Compile / exportedModule).value,
+        (`persistance` / Compile / exportedModule).value,
```

Comment:
`os-environment` now contains support for passing `Message`s via `Channel`. Messages serde is via `persistance` - hence the dependency.